r/R/schema.R

```diff
@@ -121,7 +121,25 @@ Schema <- R6Class("Schema",
     Equals = function(other, check_metadata = FALSE, ...) {
       inherits(other, "Schema") && Schema__Equals(self, other, isTRUE(check_metadata))
     },
-    export_to_c = function(ptr) ExportSchema(self, ptr)
+    export_to_c = function(ptr) ExportSchema(self, ptr),
+    code = function() {
+      names <- self$names
+      codes <- map2(names, self$fields, function(name, field){
+        withCallingHandlers(
+          field$type$code(),
+          error = function(cnd) {
+            abort(
+              glue::glue('Error getting code for field "{name}"'),
```

Comment:
I think this (and the other one) might need to be a `paste()` as `glue` isn't a package we depend on currently.

Reply:
oops. will fix.